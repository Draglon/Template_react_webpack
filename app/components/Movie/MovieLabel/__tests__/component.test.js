import React from 'react';

import MovieLabelComponent from '../component';

describe('<MovieLabelComponent />', () => {
  const props = {
    title: 'some title',
    text: 'some text',
  };
  const component = shallow(<MovieLabelComponent {...props} />);

  it('should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
