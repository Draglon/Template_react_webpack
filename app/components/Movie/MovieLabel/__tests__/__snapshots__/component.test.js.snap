// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieLabelComponent /> should match snapshot 1`] = `
<Row
  gutter={0}
>
  <Col
    offset={2}
    span={20}
  >
    <Paragraph
      className="movie__text"
    >
      <b>
        some title
        : 
      </b>
      <span>
        some text
      </span>
    </Paragraph>
  </Col>
</Row>
`;
